app/controllers/api/v1/ping_controller.rb

```diff
@@ -0,0 +1,7 @@
+class Api::V1::PingController < ApiController
+  skip_before_action :ensure_jwt
+
+  def show
+    render json: { timestamp: Time.now.to_i }
```

Comment:
Good idea to have a ping endpoint as we can tie our pingdom alert to this once the API is in use by Volt.